filebeat/module/elasticsearch/jvm/test/gc.jdk8.624.log

```diff
@@ -0,0 +1,702 @@
+2018-06-11 01:51:14 GC log file created /var/log/elasticsearch/gc.log.4
```

Comment:
As this log contains `jdk` in the name I'm now thinking if this is actually a jdk log and not jvm. I'm confused :-)

Reply:
This log came from a running JDK (as opposed to JRE) and should be renamed to `gc.log`.